test/jtreg-ext/requires/VMProps.java

```diff
@@ -417,22 +416,6 @@ protected String vmHasDTrace() {
         return "" + WB.isDTraceIncluded();
     }
 
-    /**
-     * @return "true" if compiler in use supports RTM and "false" otherwise.
-     * Note: Lightweight locking does not support RTM (for now).
-     */
-    protected String vmRTMCompiler() {
```

Comment:
[JDK-8358542](https://bugs.openjdk.org/browse/JDK-8358542) exists to remove this so you would need to add that bug id to this PR. However, it seems the bug management for this has gotten completely messed up so you may need to scrap this PR and file a new bug and PR for this part.

Reply:
Since you added issue JDK-8358542, can you also remove the function under this: vmRTMCPU - that function isn't referenced anywhere either.